lib/system.nim

```diff
@@ -267,6 +267,7 @@ const ThisIsSystem = true
 
 proc internalNew*[T](a: var ref T) {.magic: "New", noSideEffect.}
   ## Leaked implementation detail. Do not use.
+  # TODO: obsolete; remove after the next csource update
```

Comment:
hmm, did you want to add a check for sym name in magics? I believe that'll allow detection of whether it was an `internalNew` call and that could be an assert (prevent regressions).

Reply:
I went with wrapping the definition in a check for whether `refc` is used. That way, the symbol is only available during the first bootstrap iteration.